logstash-core/src/main/java/org/logstash/instrument/witness/MetricSerializer.java

```diff
@@ -36,7 +36,7 @@
          */
         static MetricSerializer<Metric<Long>> longSerializer(JsonGenerator gen) {
             return m -> {
-                if (m != null && m.isDirty() && m.getValue() != null) {
+                if (m != null && m.getValue() != null) {
```

Comment:
...actually since this is currently dead code (no paths to execute it), i will fix this on the next PR that will also include the PQ and DLQ stuff.  (My working copy already has those and it is making my manual merge difficult).